charts/prometheus/README.md

```diff
@@ -2,57 +2,82 @@
 
 [Prometheus](https://prometheus.io/), a [Cloud Native Computing Foundation](https://cncf.io/) project, is a systems and service monitoring system. It collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts if some condition is observed to be true.
 
-## TL;DR;
-
-```console
-helm install stable/prometheus
-```
-
-## Introduction
-
 This chart bootstraps a [Prometheus](https://prometheus.io/) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
 ## Prerequisites
 
 - Kubernetes 1.3+ with Beta APIs enabled
 
-## Installing the Chart
+## Get Repo Info
 
-To install the chart with the release name `my-release`:
+```console
+$ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
```

Comment:
the `$` shouldn't be there if there isn't a real output after exec-ing the cmd.

Reply:
The same for all the `$` additions.

Reply:
I just added for consistency between all console blocks, so it's unambiguous to end users which parts of codeblocks are commands, and which refer to output or otherwise. Does that makes sense?

Reply:
Yeah - but if there's no output, there should be no `$` according to the MD linter.

Reply:
That makes sense. My feeling is, the only real reason to have multiline shell commands with no $ prefix is for multi-line copy/paste. This is how I interpret the linting rule rationale you posted above as well. I agree in this case it's a good idea to copy/paste as a block - changing! Thanks for the detailed review.